src/Components/NavBar/NotificationsBadge.tsx

```diff
@@ -64,35 +66,44 @@ export const NotificationsBadge: React.FC = () => {
           return null
         }
 
-        const trackOnHover = once(() => {
-          trackEvent({
-            subject: AnalyticsSchema.Subject.NotificationBell,
-            new_notification_count: totalUnread,
-          })
-        })
-
         return (
-          <Box onMouseOver={trackOnHover}>
-            <CircularCount notifications={notifications} />
+          <Box>
+            <CircularCount
+              notifications={notifications}
+              rawCount={totalUnread}
```

Comment:
We pass along the raw count in addition to the formatted count. Analytics wants the full integer, not the formatted "99+" label string.